frontend/components/common/CompatibleDevice.tsx

```diff
@@ -4,23 +4,27 @@ import { Product } from '@models/product';
 
 export type CompatibleDeviceProps = {
    device: NonNullable<Product['compatibility']>['devices'][number];
-   truncate?: number;
+   maxVisibleModels?: number;
 };
 
 export function CompatibleDevice({
    device,
-   truncate = 0,
+   maxVisibleModels = 0,
 }: CompatibleDeviceProps) {
    const variants = React.useMemo(
       () => device.variants.filter(Boolean),
       [device.variants]
    );
    const [visibleVariants, hiddenVariantCount] = React.useMemo(
       () =>
-         truncate > 0 && variants.length > truncate
-            ? [variants.slice(0, truncate), variants.length - truncate]
+         // If there are `max + 1` variants, show the last variant.
+         maxVisibleModels > 0 && variants.length > maxVisibleModels + 1
```

Comment:
but isn't the max visible numbers now technically `maxVisibleModels` + 1? So the parameter isn't always accurate

Reply:
Do you have any ideas as to what we should call it? I was kind of at a loss for a clear name

Reply:
We could do `maxModelLines` and restore the previous `-1` check.

Reply:
I honestly think `truncate` is _ok_ just not imo the best. I think `maxVariantsShown` could indicate, but basically i think theres two options here.

1.) We go to the old -1 way and the variable indicate the maximum lines under the specific devices section.
2.) We go with the current way and the variable indicates the maximum lines of variants (so the compatibility link will be +1 to the line count) under the specific device sections.

Reply:
Yea I agree with you. See what you think about https://github.com/iFixit/react-commerce/pull/1183/commits/9ce8d42e2dbface16f5fbf209c42a67d81d46f94.  The +1, -1 logic is to handle the case where the # of variants === max lines to show.